### ISSUE TITLE
[buildbots] compiler updates (Trac #1819)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1811]
On the last software call it was decided that we'd open up production projects for C++11 by September 5th.  We'll need the compilers updated to support C++11 on any bots with old compilers.  We know the SL6 bot needs this upgrade at the very least.

Migrated from https://code.icecube.wisc.edu/ticket/1819
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:38", 
    "description": "On the last software call it was decided that we'd open up production projects for C++11 by September 5th.  We'll need the compilers updated to support C++11 on any bots with old compilers.  We know the SL6 bot needs this upgrade at the very least.", 
    "reporter": "olivas", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067158057333", 
    "component": "infrastructure", 
    "summary": "[buildbots] compiler updates", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2016-08-14T17:22:05", 
    "milestone": "", 
    "owner": "nega", 
    "type": "task"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160815T20:15:51: 
* nega commented:

> In changeset 2649:
> 
> ```text
> #!CommitTicketReference repository="IceTray" revision="2649"
> enable C++11
> 
> Turn on C++11 support via the `-std=c++14` flag. This also gets us thinking
> ahead to C++14. Additionally, specify `-std=gnu89` for C files. (This can
> be overridden for c99/gnu99 on a per file/project basis too.) We also ban
> non-C++11-conformant compilers with this patch.
> 
> As an external support activity the buildbots are now all using C++11
> conformant compilers. SL6 bots are using clang 3.4 as provided by the EPEL.
> 
> closes #1811
> ```


* nega changed resolution from "" to "fixed"
* nega changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190112T00:24:16: nwhitehorn commented:

> In changeset 170314:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="170314"
> enable C++11
> 
> Turn on C++11 support via the `-std=c++14` flag. This also gets us thinking
> ahead to C++14. Additionally, specify `-std=gnu89` for C files. (This can
> be overridden for c99/gnu99 on a per file/project basis too.) We also ban
> non-C++11-conformant compilers with this patch.
> 
> As an external support activity the buildbots are now all using C++11
> conformant compilers. SL6 bots are using clang 3.4 as provided by the EPEL.
> 
> closes #1811
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:38: 
* olivas commented:

> Milestone August 2016 deleted


* olivas changed milestone from "August 2016" to ""
